config/critical_path_css.yml

```diff
@@ -0,0 +1,22 @@
+defaults: &defaults
+  # If using the asset pipeline, provide the manifest name
+  manifest_name: application
+  # Else provide the relative path of your CSS file from the /public directory
+  # css_path: /path/to/css/from/public/main.css
+  # Or provide a separate path for each route
+  # css_paths:
+  #   - /path/to/css/from/public/main.css
+  routes:
+    - /
+
+development:
+  <<: *defaults
+  base_url: http://localhost:3000
+
+staging:
```

Comment:
Should we also make one for stable.publiclab.org?

Reply:
Perhaps we can just make this stable instead? Or would that mean it'd stop working in unstable?  Can we have both, or a wildcard like `*.publiclab.org`? Thank you!!!

Reply:
The wildcard * is not working so I have modified it to stable :sweat_smile: